_data/clients/beagleim.yml

```diff
@@ -1,4 +1,7 @@
-name: Beagle IM
+name: Beagle IM by Tigase
```

Comment:
Is this really the full title? https://beagle.im/ looks like it is only "Beagle IM"

Reply:
Sometimes we go for shorthand "BeagleIM" but the full name is `Beagle IM by Tigase, Inc.`. I've pushed updated revision to include missing bit

Reply:
Okay then. :)